addusers.py

```diff
@@ -137,12 +137,15 @@ def create_user(self, fullname, username, password, description, email, project_
             
             setpass_token = self.setpass.get_token(user.id, password, pin)
             password_url = self.setpass.get_url(setpass_token)
-
-            usr_cfg = dict(config.items('welcome_email'))
-            pwd_cfg = dict(config.items('password_email'))
-
+        
+            usr_cfg = email_defaults.copy() 
```

Comment:
I would make an ``email.py`` module which takes config and returns the appropriate mail template based on it.